src/frontend/src/optimizer/plan_node/stream_index_scan.rs

```diff
@@ -72,19 +72,17 @@ impl fmt::Display for StreamIndexScan {
         let verbose = self.base.ctx.is_explain_verbose();
         let mut builder = f.debug_struct("StreamIndexScan");
 
-        builder
-            .field("index", &format_args!("{}", self.logical.table_name()))
-            .field(
-                "columns",
-                &format_args!(
-                    "[{}]",
-                    match verbose {
-                        false => self.logical.column_names(),
-                        true => self.logical.column_names_with_table_prefix(),
-                    }
-                    .join(", ")
-                ),
-            );
+        builder.field("index", &self.logical.table_name()).field(
+            "columns",
+            &format!(
```

Comment:
I think `format_args!` is preferable to `format!` here because it avoids allocation. Correct me if I'm wrong.